docs/source/EccFramework.rst

```diff
@@ -0,0 +1,154 @@
+ECC Framework
+======================================================================================================
+
+The QECC library offers means for automatic implementation and evaluation of error-correcting codes for quantum
+computing. More precisely, the library allows to automatically apply different error correction schemes to quantum
+circuits provided as OpenQASM files or Qiskit :code:QuantumCircuit objects. The "protected" quantum circuits can then be exported again in the form of OpenQASM files or can be directly used for noise-aware quantum circuit simulation. For the latter case, we also provide a wrapper script which makes use of the provided framework to apply error correction schemes to circuits and directly simulate those circuits using qiskit.
```

Comment:
You need to insert backticks after the tag (`:code:`) and at the end of the thing you want to tag. Please inspect the documentation output for cases where this is missing and ugly strings are printed including the tags.

Reply:
I changed it to "QuantumCircuit"